widget/playground/src/App.tsx

```diff
@@ -2,6 +2,7 @@ import { Widget, WidgetProvider } from '@rango-dev/widget-embedded';
 import React, { useEffect } from 'react';
 import { Route, Routes } from 'react-router-dom';
 
```

Comment:
Instead of branching the main project, why did you fork it?

Reply:
At first, I didn't have the write permission on the repo so I forked. From now on I will make branches from the main repo.